### PR TITLE
PRM Automated Test - LPS-172458 master

### PR DESCRIPTION
**Pull request by juliosilveira77**
https://issues.liferay.com/browse/LPS-172458

### CODE REVIEW

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase [hunk 1]
```diff
@@ -35,6 +35,20 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172458. User can access Deal Registration Listing through Quick Links"
+	@priority = "4"
+	test CanAccessDealListingViaQuickLinks {
+		task ("When I click on 'MDF' button, on Quick Links section") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase:41]: Hi @juliosilveira77, could you please rename this task to: "When I click on 'Deal Registration' button, on Quick Links section"?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase [hunk 2]
```diff
@@ -35,6 +35,20 @@ definition {
...
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "deal-registrations",
+				buttonText = "Deal Registration");
+		}
+
+		task ("Then I'm redirected to the 'MDF Request Listing' page") {
+			PRMPages.assertPageTitle(pageTitle = "Partner Deal Registration");
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase:48]: Hi @juliosilveira77, could you please rename this task to: "Then I'm redirected to the 'Deal Registration Listing' page"?

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
Hi @juliosilveira77,I see that you have some duplicated commits, that aren't necessary. Could you please do a git reset e redo your commits? Thanks!

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:forward

**Pull request closed without merging**